src/settings.js

```diff
@@ -805,14 +805,14 @@ var NODEJS_CATCH_REJECTION = true;
 // Whether to support async operations in the compiled code. This makes it
 // possible to call JS functions from synchronous-looking code in C/C++.
 //
-// - Run binaryen's Asyncify pass to transform the code using asyncify. This
-//   emits a normal wasm file in the end, so it works everywhere, but it has a
-//   significant cost in terms of code size and speed.
+// - 'binaryen': Run binaryen's Asyncify pass to transform the code using
+//   asyncify. This emits a normal wasm file in the end, so it works everywhere,
+//   but it has a significant cost in terms of code size and speed.
 //   See https://emscripten.org/docs/porting/asyncify.html
-// - Depend on VM support for the wasm stack switching proposal. This allows
-//   async operations to happen without the overhead of modifying the wasm.
-//   This is experimental atm while spec discussion is ongoing, see
-//   https://github.com/WebAssembly/js-promise-integration/
+// - 'jspi': Depend on VM support for the JavaScript Promise Integration
+//   proposal. This allows async operations to happen without the overhead of
+//   modifying the wasm. This is experimental atm while spec discussion is
+//   ongoing, see https://github.com/WebAssembly/js-promise-integration/
```

Comment:
Should we mention which on is the default if you just so `-sASYNCIFY` (I assume its binaryen)?